src/pybind/mgr/orchestrator/module.py

```diff
@@ -1148,24 +1148,50 @@ def _process_cert_store_json(self, d: Dict[str, Any], level: int = 0) -> str:
         result_str = ''
         indent = '  ' * level
         for k, v in d.items():
-            if isinstance(v, dict):
+            if isinstance(v, dict) and v:
                 result_str += f'{indent}{k}\n'
                 result_str += self._process_cert_store_json(v, level + 1)
-            else:
-                result_str += f'{indent}{k} - {v}\n'
+            elif v or isinstance(v, int):
+                result_str += f'{indent}{k}: {v}\n'
         return result_str
 
-    @_cli_read_command('orch cert-store cert ls')
-    def _cert_store_cert_ls(self, format: Format = Format.plain) -> HandleCommandResult:
-        completion = self.cert_store_cert_ls()
+    @_cli_read_command('orch certmgr reload')
+    def _cert_store_reload(self, format: Format = Format.plain) -> HandleCommandResult:
+        completion = self.cert_store_reload()
+        output = raise_if_exception(completion)
+        return HandleCommandResult(stdout=output)
+
+    @_cli_read_command('orch certmgr cert ls')
+    def _cert_store_cert_ls(self, show_details: bool = False, format: Format = Format.plain) -> HandleCommandResult:
+        completion = self.cert_store_cert_ls(show_details)
         cert_ls = raise_if_exception(completion)
         if format != Format.plain:
             return HandleCommandResult(stdout=to_format(cert_ls, format, many=False, cls=None))
         else:
             result_str = self._process_cert_store_json(cert_ls, 0)
             return HandleCommandResult(stdout=result_str)
 
-    @_cli_read_command('orch cert-store key ls')
+    @_cli_read_command('orch certmgr entity ls')
+    def _cert_store_entity_ls(self, format: Format = Format.plain) -> HandleCommandResult:
+        completion = self.cert_store_entity_ls()
+        entity_ls = raise_if_exception(completion)
+        if format != Format.plain:
+            return HandleCommandResult(stdout=to_format(entity_ls, format, many=False, cls=None))
+        else:
+            result_str = yaml.dump(entity_ls, default_flow_style=False, sort_keys=False)
+            return HandleCommandResult(stdout=result_str)
+
+    @_cli_read_command('orch certmgr cert check')
```

Comment:
Hi @rkachach ,
On running this command, it returns nothing if there are no expiring/invalid certificates.
Can we add some message for the users as we discussed? 
Thanks!

Reply:
Hi @sayaleeraut, yeah, I think it's more user-friendly to show some success message. I'll change the code :+1: